code/__HELPERS/turfs.dm

```diff
@@ -207,6 +207,11 @@ Turf and target are separate in case you want to teleport some distance from a t
 	if(!istype(checked_atom))
 		return
 
+	//Find coordinates
+	var/turf/atom_turf = get_turf(checked_atom) //use checked_atom's turfs, as it's coords are the same as checked_atom's AND checked_atom's coords are lost if it is inside another atom
+	if(!atom_turf)
+		return null
```

Comment:
why?

Reply:
cant remember i need to stop doing this. was probably for the early return